### PR TITLE
sha3sum 1.1.5 (new formula)

### PR DESCRIPTION
**Pull request by zbeekman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by zbeekman** [comment]
FYI @maandree, @danteu, @nremond, @cleishm: `libkeccak` has been added to homebrew-core and this PR will add `sha3sum`. To port these to linuxbrew I'll need to figure out which of the build and runtime dependencies listed in the respective `DEPENDENCIES` file in each repo will need to be installed through linuxbrew; probably all of them.

File: Formula/sha3sum.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  desc "Keccak, SHA-3, SHAKE, and RawSHAKE checksum utilities"
+  homepage "https://github.com/maandree/sha3sum"
+  url "https://github.com/maandree/sha3sum/archive/1.1.5.tar.gz"
+  sha256 "1a4eef5b09b7f70af1f6970840475d4735a14b4fab21937b9fd104b8606654ce"
+
+  depends_on "libkeccak"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+    inreplace "test", "./", ""
```
Comment (sjackman) [inline Formula/sha3sum.rb]:
```suggestion
    inreplace "test", "./", "#{bin}/"
```

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Formula/sha3sum.rb (hunk 1 above)
Comment (zbeekman) [reply Formula/sha3sum.rb]: Good call!

**Review by sjackman — APPROVED** [review]

**Comment by nremond** [comment]
That is really great!
Thank you

**Comment by zbeekman** [comment]
> To port these to linuxbrew I'll need to figure out which of the build and runtime dependencies listed in the respective `DEPENDENCIES` file in each repo will need to be installed through linuxbrew; probably all of them.

It looks like all build and runtime deps are actually included by default with linuxbrew, so the next time homebrew-core is pulled into Linuxbrew these will get picked up, and should just work™️

**Pull request closed without merging**